go.mod

```diff
@@ -103,6 +103,7 @@ replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	github.com/openshift/api => github.com/openshift/api v3.9.0+incompatible
 	github.com/openshift/client-go => github.com/openshift/client-go v3.9.0+incompatible
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
```

Comment:
please change also here
https://github.com/k8snetworkplumbingwg/kubemacpool/pull/367/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R12

Reply:
I don't follow, can you be specific what change is needed? (AFAIU go tidy go vendor should generate the change..)

Reply:
you can pin the version on the require as well i think ? not just on the replace (it is fine to keep it on the replace as well)
so if you can update the version of `github.com/prometheus/client_golang v1.11.0` go.mod line 12 as well ?

Reply:
ah.. why is it better to do so?

Reply:
just feels more straight forward imho, instead of have two different versions in the file when we don't have to
unless you are against

Reply:
except go.mod all the files are generated, so this is fine.